portfolio/src/main/java/com/google/sps/data/Marker.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+/** Represents a marker on the map. */
+public class Marker {
```

Comment:
It is near to the end of the week 5, so you don't have to take my recommendation for this project.

For java value/data class, you can try the AutoValue library to auto generate these boilerplate code:
https://github.com/google/auto/blob/master/value/userguide/index.md

Reply:
I hadn't heard about AutoValue before, I will look into it for creating value classes in the capstone. Thank you!